webpack.config.development.js

```diff
@@ -79,11 +121,28 @@ export default validate(merge(baseConfig, {
      */
     new webpack.DefinePlugin({
       'process.env.NODE_ENV': JSON.stringify('development')
-    })
+    }),
+    // turn debug mode on.
+    new webpack.LoaderOptionsPlugin({
+      debug: true
+    }),
   ],
 
   /**
    * https://github.com/chentsulin/webpack-target-electron-renderer#how-this-module-works
    */
-  target: 'electron-renderer'
-}));
+  target: 'electron-renderer',
+  devServer: {
+    port,
+    hot: true,
+    inline: false,
+    historyApiFallback: true,
+    contentBase: path.join(__dirname, 'dist'),
+    publicPath,
+    setup() {
+      spawn('npm', ['run', 'start-hot'], { shell: true, env: process.env, stdio: 'inherit' })
```

Comment:
Maybe just `"dev": "cross-env START_HOT=1 npm run hot-server"` and check `process.env.START_HOT` will be cleaner.